src/viam/examples/project/pkg-config/Makefile

```diff
@@ -1,19 +1,24 @@
 CXX = c++
 
-CXXFLAGS = -std=c++14
-CXXFLAGS += `pkg-config --cflags viam-cpp-sdk-libviamsdk`
-LINKFLAGS = `pkg-config --libs viam-cpp-sdk-libviamsdk`
+VIAM_PREFIX ?= ../../../../../build/install
+PKG_CONFIG_PATH ?= $(VIAM_PREFIX)/lib/pkgconfig
+
+
+CXXFLAGS = -std=c++17
+CXXFLAGS += `PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --cflags viam-cpp-sdk-libviamsdk`
+LINKFLAGS = `PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --libs viam-cpp-sdk-libviamsdk`
+LDFLAGS ="-Wl,-rpath,$(VIAM_PREFIX)/"
```

Comment:
It is a nuisance that pkg-config doesn't propagate these. Still, I'd expect to see a `lib` on here.

Reply:
In view of our revelation about the rpath/install_path stuff, I'm going to delete this line for now and revisit this when we figure out the rpath stuff; it may obviate `VIAM_PREFIX` entirely